tests/integration.rs

```diff
@@ -4855,12 +4856,30 @@ mod common_parallel {
             .args(["--memory", format!("size={guest_memory_size_kb}K").as_str()])
             .args(["--kernel", kernel_path.to_str().unwrap()])
             .args(["--cmdline", DIRECT_KERNEL_BOOT_CMDLINE])
+            .args(["--event-monitor", format!("path={event_path}").as_str()])
             .default_disks()
             .capture_output()
```

Comment:
Please put `See: #5127` in the commit messages so that the commits will be automatically cross referenced with the issue without closing it.

Reply:
Updated commit message of each commit